lib/index.js

```diff
@@ -19,6 +19,25 @@ function spawnSeal(){
   }
 }
 
+function moveInput() {
+  window.addEventListener('keydown', function(e) {
+    if (e.keyCode === 37) {
+      narwhal.x -= 5;
+    } else if (e.keyCode === 38) {
+      narwhal.y -=5;
+    } else if (e.keyCode === 40) {
+      narwhal.y += 5;
+    } else if (e.keyCode === 39) {
+      narwhal.x += 5;
+    }
+      console.log(e.keyCode);
+  })
+}
+
+moveInput();
+
```

Comment:
This adds event listeners for the keyboard that are functional, and move the narwhal block. The animation is really choppy, so we'll have to figure out how to make things smoother, so that the narwhal is animated correctly, and moves smoothly AND quickly across the page without depending on x/y displacement for speed.